samples/apps/smallbank/app/flatbuffer_wrapper.h

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/buffer.h"
+
+#include <accounts_generated.h>
+#include <amalgamate_generated.h>
+#include <bank_generated.h>
+#include <transaction_generated.h>
+
+class FlatbufferSerializer
+{
+protected:
+  flatbuffers::FlatBufferBuilder builder;
+
+public:
+  std::unique_ptr<flatbuffers::DetachedBuffer> get_detached_buffer()
+  {
+    return std::make_unique<flatbuffers::DetachedBuffer>(builder.Release());
+  }
+
+  CBuffer get_buffer()
+  {
+    return {builder.GetBufferPointer(), builder.GetSize()};
```

Comment:
Do we need to create a new vector here? can pointers and size be used?

Reply:
It's not a vector, it is a CBuffer, i.e. pointers and vectors _are_ used